core/src/main/java/net/neoforged/fml/config/ConfigFileTypeHandler.java

```diff
@@ -31,38 +31,56 @@ public class ConfigFileTypeHandler {
     public Function<ModConfig, CommentedFileConfig> reader(Path configBasePath) {
         return (c) -> {
             final Path configPath = configBasePath.resolve(c.getFileName());
-            final CommentedFileConfig configData = CommentedFileConfig.builder(configPath).sync().
-                    preserveInsertionOrder().
-                    autosave().
-                    onFileNotFound((newfile, configFormat)-> setupConfigFile(c, newfile, configFormat)).
-                    writingMode(WritingMode.REPLACE).
-                    build();
-            LOGGER.debug(CONFIG, "Built TOML config for {}", configPath.toString());
+            final CommentedFileConfig configData = CommentedFileConfig.builder(configPath)
+                    .sync()
```

Comment:
Why was a sync added here?

Reply:
Uh? It was there [before](https://github.com/neoforged/FancyModLoader/blob/main/core/src/main/java/net/neoforged/fml/config/ConfigFileTypeHandler.java#L34), I just moved it down a bit.